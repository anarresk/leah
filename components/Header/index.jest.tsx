import React from 'react';

import {Header} from './index';
import expectShallow from '../../utils/testing/expectComponent';

describe('components/Header', () => {
    it('renders as expected', () => {
        expectShallow(<Header />).toMatchSnapshot();
    });
});
