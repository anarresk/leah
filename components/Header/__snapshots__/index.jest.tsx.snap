// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header renders as expected 1`] = `
<h1>
  Welcome to Sky Vegas!
</h1>
`;
